problems/removeNumbersAtOddIndices.js

```diff
@@ -1,9 +1,14 @@
 /** Remove all numbers at odd indices from an array
-* @param {number[]} arr - The input array
-* @returns {number[]} - An array removing all elements initially appearing at an odd index
-*/
-function removeNumbersAtOddIndices() {
-
+ * @param {number[]} arr - The input array
+ * @returns {number[]} - An array removing all elements initially appearing at an odd index
+ */
+function removeNumbersAtOddIndices(arr) {
+  let output = [];
+  for (let i = 0; i < arr.length; i += 2) {
```

Comment:
Try using .filter for cleaner code. Remember the callback on array methods can also take the index as a second argument. 

But I do love the step of i += 2